Assets/Project/Scripts/UnityChanControlScript.cs

```diff
@@ -33,16 +41,17 @@ public class UnityChanControlScript : MonoBehaviour
         public float jumpPower = 5.0f;
         public float animSpeed = 1.5f;
 
-        //
-        private int idleState = Animator.StringToHash("Base Layer.Idle");
-        private int locoState = Animator.StringToHash("Base Layer.Locomotion");
-        private int jumpState = Animator.StringToHash("Base Layer.Jump");
-        private int restState = Animator.StringToHash("Base Layer.Rest");
-        private int slideState = Animator.StringToHash("Base Layer.Slide");
-        private int attackState = Animator.StringToHash("Base Layer.Attack");
+        #endregion
 
-        Dictionary<int, Action> animationStates;
+        #region Ability fields
+        // animator parameters
+        public string jumpParam = "Jump";
+        public string slideParam = "Slide";
+        public string attackParam = "Attack";
+        public string deathParam = "Collision";
```

Comment:
Think these should have the const keyword then variable renamed to fit the convention
E.g:
`public const string JUMP_PARAM = "Jump"`

Reply:
cool, lets make it const, I like the params as variables as then the code is just filed with magic strings everyhwere and if you change the parameter name in the animator it all dies 😆, so I thought lets have it as variables instead